provider/openstack/firewaller.go

```diff
@@ -493,22 +461,16 @@ func (c *neutronFirewaller) DeleteMachineGroup(ctx context.ProviderCallContext,
 // UpdateGroupController implements Firewaller interface.
 func (c *neutronFirewaller) UpdateGroupController(ctx context.ProviderCallContext, controllerUUID string) error {
 	neutronClient := c.environ.neutron()
-	groups, err := neutronClient.ListSecurityGroupsV2()
-	if err != nil {
-		handleCredentialError(err, ctx)
-		return errors.Trace(err)
-	}
-	re, err := regexp.Compile(c.jujuGroupPrefixRegexp())
+	tags := []string{fmt.Sprintf("%s=%s", tags.JujuModel, c.environ.modelUUID)}
```

Comment:
Hi Adis, just curious why dont we use the controller tag in this case to query like in DeleteAllControllerGroups?